docs/INSTALL.UBUNTU.md

```diff
@@ -0,0 +1,61 @@
+Installing OVS and OVN from sources on Ubuntu
+=============================================
```

Comment:
I have seen many beginners trying to install OVS from INSTALL.md, missing a few important words and completely messing it up. This acts as a quick start guide where you can't really make a mistake.